app/src/main/java/org/mozilla/fenix/compose/Favicon.kt

```diff
@@ -38,32 +41,55 @@ fun Favicon(
     size: Dp,
     isPrivate: Boolean = false
 ) {
-    components.core.icons.Loader(
-        url = url,
-        isPrivate = isPrivate,
-        size = size.toIconRequestSize()
-    ) {
-        Placeholder {
-            Box(
-                modifier = Modifier
-                    .size(size)
-                    .background(
-                        color = when (isSystemInDarkTheme()) {
-                            true -> PhotonColors.DarkGrey30
-                            false -> PhotonColors.LightGrey30
-                        }
-                    )
-            )
+    if (inComposePreview) {
+        Placeholder(size = size)
+    } else {
+        components.core.icons.Loader(
+            url = url,
+            isPrivate = isPrivate,
+            size = size.toIconRequestSize()
+        ) {
+            Placeholder {
+                Placeholder(size = size)
```

Comment:
```suggestion
                FaviconPlaceholder(size = size)
```

Let's add a prefix for this. Having a Placeholder inside of a Placeholder is kinda disguised 🥸.